docs/continuous-integration/ci-in-datascience/working-with-notebooks/README.md

```diff
@@ -0,0 +1,124 @@
+# Data Science Pipeline
+
+As Azure DevOps doesn't allow the code reviewers to comment directly in Jupyter Notebooks, the Data Scientists(DSs) have
+to convert the notebooks to scripts before they commit and push these files to the repository.
+
+This document aims to automate this process in Azure DevOps, so the DSs don't need to execute anything locally.
+
+## Problem statement
+
+A Data Science repository has this folder structure:
+
+```bash
+
+    .
+    ├── notebooks
+    │   ├── Machine Learning Experiments - 00.ipynb
+    │   ├── Machine Learning Experiments - 01.ipynb
+    │   ├── Machine Learning Experiments - 02.ipynb
+    │   ├── Machine Learning Experiments - 03.ipynb
+    └── scripts
+       ├── Machine Learning Experiments - 00.py
+       ├── Machine Learning Experiments - 01.py
+       ├── Machine Learning Experiments - 02.py
+       └── Machine Learning Experiments - 03.py
+
+```
+
+We need this as Azure DevOps doesn't allow the Pull Request reviewers to add comments to the notebooks, so they add comments
+to the Python scripts and we apply these comments to the notebooks.
+
+We run (or used to run) this process manually before we add the files to a commit to the repository. This manual process
+can lead to errors, e.g.: I create the notebook, generate the script from it, but later I make some changes. I forgot to
+generate a new script for the changes and push asynchronous files to the repo.
```

Comment:
```suggestion
Since we have to run this process manually before we add files to a commit, this manual process is error prone, e.g. If we create a notebook, generate the script from it, but later make some changes and forget to generate a new script for the changes.
```